CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ All notable changes to this project will be documented in this file.
 ### New
 - GraphQL optimization: use single web socket to serve all subscriptions.
 - `net.find_last_shard_block` function returning account shard last block ID.
+- **Debot Module:**   
+  - Added new 3 engine routines for crypto operations and 1 routine for querying account state (balance, state type, code, data).
+  - Added debot test for engine calls.
```

Comment:
Changelog contains information about release for SDK users. No need to write about tests and and test contracts here